### DISCUSSION TITLE
Use php deployer to deploy on GCP with IAP

### DISCUSSION THREAD

**Discussion by chrey** [deployphp/deployer#3555] [Help needed]
Hello, 
Is there a way to use phpdeployer to deploy on GCP instances through IAP, meaning we have to encapsulate ssh command to use gcloud ssh compute ? 

Like ansible does here https://docs.ansible.com/ansible/latest/scenario_guides/guide_gce.html

Thanks

**Comment by antonmedv** [comment]
Currently, Deployer only works with SSH binary. But I know it's possible to get and SSH access to GCP as well (for VPS). 

For working with gcloud binary: this is a really cool idea. Will look at what is possible to do here.
